zotify/track.py

```diff
@@ -189,12 +189,17 @@ def download_track(mode: str, track_id: str, extra_keys=None, disable_progressba
 
         # a song with the same name is installed
         if not check_id and check_name:
-            c = len([file for file in Path(filedir).iterdir() if re.search(f'^{filename}_', str(file))]) + 1
+            
+            if Zotify.CONFIG.get_skip_existing():
+                Printer.print(PrintChannel.SKIPS, '\n###   SKIPPING: ' + song_name + ' (SONG ALREADY EXISTS)   ###' + "\n")
+                return
```

Comment:
this is fixed in commit https://github.com/diogosena/zotify/commit/d3eea11be3865a83017b24818ba0f240201fdfb5

Reply:
Added, thank you!